node/Dockerfile.amd64

```diff
@@ -15,6 +15,7 @@ ARG ARCH=x86_64
 ARG GIT_VERSION=unknown
```

Comment:
Does this impact non-amd64 architectures as well? I suspect so.

May still be worth merging this increment without those, since it's strictly better to have one functioning architecture than none :grin:

Reply:
Yeah, agree.